lib/vdsm/network/link/setup.py

```diff
@@ -55,3 +74,15 @@ def parse_nets_bridge_opts(nets):
                 opt.split('=', 1) for opt in opts_str.split(' ')
             )
             yield name, bridge_opts
+
+
+def parse_nets_bridge_opts_for_nmstate(opts):
+    if opts:
+        opts_str = opts.replace('_', '-')
+        bridge_opts = dict(
+            opt.split('=', 1) for opt in opts_str.split(' ')
+        )
+        for key in bridge_opts.keys():
+            bridge_opts[key] = nmstate_options_types[key](eval(bridge_opts[key]))
```

Comment:
Why is the `eval` needed?

Reply:
because `bridge_opts` values are strings and nmstate seem to accept int/bool

Reply:
Having eval on user input is huge security risk, if `bool` is not working we need to create our own function that will do the conversion.